### PR TITLE
Add references to issues and PRs in markdown

### PR DESCRIPTION
**Pull request by MrBartusek**
This PR adds [linkify-issues](https://github.com/sindresorhus/linkify-issues) library which support notations like `#477` or `hacs/frontent#477`

See last change with Spanish translation:

Before:

image

After:

image

### CODE REVIEW

**Review by ludeeus — CHANGES_REQUESTED** [review]
This adds another dependency (which again drags in 7 more), that is too much for something that can be handled without a third-party library.

**Comment by MrBartusek** [comment]
@ludeeus Done!

image

File: src/tools/markdown/markdown.ts [hunk 1]
```diff
@@ -65,6 +65,16 @@ export class markdown {
       return `[\`${hash}\`](${url})`;
     });
 
+    // Add references to issues and PRs
+    if (repo != undefined) {
+      input = input.replace(/(?:(?<![/\w-.])\w[\w-.]+\/\w[\w-.]+|\B)#[1-9]\d*\b/g, (reference) => {
+        const fullReference = reference.replace(/^#/, `${repo.full_name}#`);
+	      const [fullName, issue] = fullReference.split('#');
+        const url = `https://github.com/${fullName}/issues/${issue}`;
```
Comment (ludeeus) [inline src/tools/markdown/markdown.ts:73]:
```suggestion
        const fullReference = reference.replace(/^#/, `${repo.full_name}#`);
        const [fullName, issue] = fullReference.split('#');
        const url = `https://github.com/${fullName}/issues/${issue}`;
```

**Review by ludeeus — APPROVED** [review]
Awesome!

**Pull request merged**